### ISSUE TITLE
flash memory 

### ISSUE THREAD

**Issue by Tahe97** [afiskon/stm32-fatfs-examples#1]
can we use fatfs library with the flash memry of our stm32 . if yes what should we put in the USERPath ?

**Comment by exxecute** [comment]
Yes, we can use fatfs with flash, but also that not good at all ;/
cause of how stm32 flash works and fatfs works, so erasing/writing can take up seconds.
